### PR TITLE
style(frontend): reorder items in user popover on main page

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

As recent requirement, we were asked to re-order the list of items in the menu popover on the main page.

## Before

Screenshot 2024-07-17 at 11 28 48

## After

Screenshot 2024-07-17 at 14 10 05

### CODE REVIEW

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/core/Menu.svelte [hunk 1]
```diff
@@ -52,6 +52,18 @@
 
 		<Hr />
 
+		<ButtonMenu ariaLabel={$i18n.navigation.alt.more_settings} on:click={gotoSettings}>
```
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Menu.svelte:55]: I am not really sure how would you modify it. You mean the ariaLabel shall become the link to the Settings page?

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/core/Menu.svelte:55]: But no rush, the PR just reminded me it.

[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Menu.svelte:55]: Alas, with:

- PR #1991 

We found out that inside the popover, a <a> tag would refresh the entire page (more details in the PR)

**Pull request merged**